tests/components/awair/fixtures/local_devices.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
So annoying it doesn't have the name in it. We should reach out to Awair to request this.

Reply:
Agreed, its unfortunate